src/service/chat.js

```diff
@@ -143,9 +143,11 @@ const sendSignedDMOtherProfile = async ({user_id, message}) => {
   return Promise.reject(response.data?.data);
 };
 
-const getOrCreateAnonymousChannel = async (userId) => {
+const getOrCreateAnonymousChannel = async (userId, oldChannelId = null, context = null) => {
   const payload = {
-    members: [userId]
+    members: [userId],
+    oldChannelId,
+    context
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `getOrCreateAnonymousChannel` now accepts two additional optional parameters: `oldChannelId` and `context`. Ensure that all calls to this function are updated accordingly, if necessary. Also, it's important to validate these inputs before using them in the payload.

```diff
const getOrCreateAnonymousChannel = async (userId, oldChannelId = null, context = null) => {
   const payload = {
    members: [userId],
    oldChannelId,
    context
   };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->